apps/astarte_appengine_api/lib/astarte_appengine_api/groups/grouped_device.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Astarte.AppEngine.API.Groups.GroupedDevice do
+  use TypedEctoSchema
+  alias Astarte.AppEngine.API.UUID
+
+  @primary_key false
+  typed_schema "grouped_devices" do
+    field :group_name, :string
+    field :insertion_uuid, UUID
+    field :device_id, UUID
+  end
```

Comment:
The primary key defined for the table `((group_name), insertion_uuid, device_id)`, so we can specify `primary_key: true` on all columns instead of leaving the schema without a primary key `@primary_key false`